### ISSUE TITLE
SSLError on trying to get players list

### ISSUE THREAD

**Issue by smani6** [MichaelSnowden/OpenDota-Python-Client#1]
Hi, thanks for the repo!! but I am not able to use any opendota api's as I am getting urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.opendota.com', port=443): Max retries exceeded with url: /api/matches/3502105825 (Caused by SSLError(SSLError(1, u'[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:590)'),))

Can you please say me how to fix the above issue ?? do i need to pass API_Key ??

**Comment by MichaelSnowden** [comment]
Can you post the code you used?

**Comment by smanikandan7** [comment]
from __future__ import print_statement
import time
import swagger_client
from swagger_client.rest import ApiException
from pprint import pprint

# create an instance of the API class
api_instance = swagger_client.ProPlayersApi()

try: 
    # GET /proPlayers
    api_response = api_instance.pro_players_get()
    pprint(api_response)
except ApiException as e:
    print("Exception when calling ProPlayersApi->pro_players_get: %s\n" % e)

I installed all the dependencies and used the above code to get pro players - and which raised the above exception.

It seems we need to authenticate ?? If so can you please tell me how to pass API Key.

**Comment by MichaelSnowden** [comment]
Sorry, I can't recreate this bug. You shouldn't need an API key

**Comment by MichaelSnowden** [comment]
Sounds like this isn't a problem with the client, something wrong with your SSL stuff, maybe this: https://stackoverflow.com/questions/35405092/sslerror-sslv3-alert-handshake-failure ?

**Comment by smanikandan7** [comment]
Yes figured it out some problem with my ssl, thanks for your comments !! and thanks for the repo !!
